crates/fluvio-connector-package/src/context.rs

```diff
@@ -0,0 +1,11 @@
+use std::collections::HashMap;
+
+use serde::Serialize;
+
+/// Context for the template engine
+/// This is the data that is available to the template engine
+/// when it is rendering the template.
+#[derive(Serialize)]
+pub(crate) struct Context {
```

Comment:
Maybe we should put this type into `renderer` module, wdyt? It is used only there and seems like impl detail specific only to renderer while occupies such a common module name as `context`.